pkg/export/export.go

```diff
@@ -475,12 +484,66 @@ func (e *Exporter) triggerNext() {
 	}
 }
 
-// ClientName and Version are used to identify to User Agent. TODO(maxamin): automate versioning.
 const (
+	// ClientName is used to identify the User Agent.
 	ClientName = "prometheus-engine-export"
-	Version    = "0.6.3"
+	// Align with git tag.
+	// TODO(TheSpiritXIII): Remove with https://github.com/golang/go/issues/50603
+	exportVersion = "v0.7.0-rc.0"
```

Comment:
sounds like Version can give different version then this - can we elaborate what version it is then with comment or var name? Default? Rule Eval one?

Reply:
Looks like it is a main module version (not a version used if export is a dependency)

Reply:
Renamed to `mainModuleVersion`.